test/scenarios.py

```diff
@@ -10145,7 +10145,7 @@
         },
     ),
     Scenario(
-        "scenario-analytics-1",
+        "analytics-1",
```

Comment:
I really like the idea of splitting analytics tests into a separate job

There is still a question for scenarios `26-1` and `26-2`, they are also testing analytics events, though they also have some other functional testing alongside it, i would lean towards moving them to the `analytics` space as well, what is your opinion?

Reply:
Yeah, sure. Good point :+1:

Reply:
Moved